src/commands/manage/ban.js

```diff
@@ -18,31 +18,49 @@ module.exports = class Ban {
         .setRequired(false)
       );
   }
-  
-  run(interaction) {
-    if (interaction.user.id !== OWNER && !ADMIN.includes(interaction.user.id)) return;
 
-    const client = interaction.client;
+  async run(interaction) {
+    let errorEmbed = new EmbedBuilder().setColor(getColor(0));
+    // if (interaction.user.id !== OWNER && !ADMIN.includes(interaction.user.id)) {
```

Comment:
Better uncomment this so people cant abuse it

Reply:
tmrw